python/ql/src/semmle/python/types/FunctionObject.qll

```diff
@@ -56,26 +56,28 @@ abstract class FunctionObject extends Object {
 
     /**
      * Gets the `ControlFlowNode` that will be passed as the nth argument to `this` when called at `call`.
-     * This predicate will correctly handle `x.y()`, treating `x` as the zeroth argument.
+     *        This predicate will correctly handle `x.y()`, treating `x` as the zeroth argument.
```

Comment:
what is up with these whitespace changes?

Reply:
Probably conflict resolution where I'm taking the local version instead of from master, like the whitespace below.

Reply:
Done!